app/services/team_deletion_service.rb

```diff
@@ -21,10 +21,16 @@ def call
 
   private
 
+  # @note (Mohammed Seedat 2019-04-03)
+  #   Deactivation information is stored in Team name string for reporting
+  #   ease purposes. Note use of brackets to allow extraction of team name
+  #   without additional information when required.
   def soft_delete
-    @team.update_attributes!(deleted_at: Time.current,
-                             name: "DEACTIVATED #{@team.name} " + Time.now.to_s)
-  end
-
+    deletion_date = DateTime.now.strftime('%F %T')
 
+    @team.update_attributes!(
```

Comment:
I think we need a simpler design. As we know the team is deleted, we could simply override the team_name property for a deleted team rather than actually updating it in the DB, and then trying to parse it out again.

Reply:
Out of scope